skills/declarative/calendarSkill/language-generation/en-us/icons.en-us.lg

```diff
@@ -53,4 +53,10 @@
 
 # TitleIconUrl
-- data:image/svg+xml,%3Csvg width='8' height='12' viewBox='0 0 28 35' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M24.375 1.875H30V28.125H0V1.875H5.625V0H7.5V1.875H22.5V0H24.375V1.875ZM5.625 3.75H1.875V7.5H28.125V3.75H24.375V5.625H22.5V3.75H7.5V5.625H5.625V3.75ZM1.875 26.25H28.125V9.375H1.875V26.25Z' fill='%23333333'/%3E%3C/svg%3E%0A
\ No newline at end of file
+- data:image/svg+xml,%3Csvg width='8' height='12' viewBox='0 0 28 35' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M24.375 1.875H30V28.125H0V1.875H5.625V0H7.5V1.875H22.5V0H24.375V1.875ZM5.625 3.75H1.875V7.5H28.125V3.75H24.375V5.625H22.5V3.75H7.5V5.625H5.625V3.75ZM1.875 26.25H28.125V9.375H1.875V26.25Z' fill='%23333333'/%3E%3C/svg%3E%0A
+
+# AttendeeIconUrl
```

Comment:
There is already a ContactIconUrl that is the same as this one. We should use "Contact" labelling instead of "Attendee" because attendee implies that person will attend a meeting, but that is not always the case.

Reply:
fixed